pixi.toml

```diff
@@ -0,0 +1,168 @@
+[project]
+name = "datajudge"
+channels = ["conda-forge"]
+platforms = ["linux-64", "osx-arm64", "win-64"]
+
+[tasks]
+postinstall = "pip install --no-build-isolation --no-deps --disable-pip-version-check -e ."
+
+[dependencies]
+python = ">=3.8"
+scipy = "*"
+sqlalchemy = ">=1.4,<3"
+flit-core = "*"
+flit = "*"
```

Comment:
i'd say that `flit` and `flit-core` are "build" dependencies that should probably go to a `dev` feature of sorts.

Reply:
let's add them to `[host-dependencies]`. we talked about this earlier as well

Reply:
yes, that should work

Reply:
35ab3ab